### PR TITLE
remove not needed custom Postgres dialect

### PR DESCRIPTION
**Pull request by biergit**
`PostgreSQL10Dialect` has been removed from Hibernate and reading `Lob`s seems to work fine with the default settings. Tested with the provided Postgres 15 image.

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactory.java [hunk 1]
```diff
@@ -1,21 +1,21 @@
 package tech.jhipster.lite.generator.server.springboot.database.postgresql.domain;
 
-import static tech.jhipster.lite.generator.server.springboot.database.sqlcommon.domain.SQLCommonModuleBuilder.*;
-import static tech.jhipster.lite.module.domain.JHipsterModule.*;
+import static tech.jhipster.lite.generator.server.springboot.database.sqlcommon.domain.SQLCommonModuleBuilder.sqlCommonModuleBuilder;
```
Comment (DamnClin) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactory.java:3]: Can you set your static import limit to 1 in your IDE and reorganize imports in this file?

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java [hunk 2]
```diff
@@ -1,7 +1,8 @@
 package tech.jhipster.lite.generator.server.springboot.database.postgresql.domain;
 
 import static org.mockito.Mockito.when;
-import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.*;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.assertThatModuleWithFiles;
```
Comment (DamnClin) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java:4]: Can you set your static import limit to 1 in your IDE and reorganize imports in this file?

**Review by DamnClin — COMMENTED** [review]
I'm totally for this, @pascalgrimaud do you agree with this change?

**Comment by DamnClin** [comment]
Looks like you did not run prettier. You can install the git hook (so commit will trigger a formatting) by doing `npm i` and, for this one, to fix the problems you can do `npm run prettier:format`

**Comment by biergit** [comment]
@DamnClin that doesn't work for me:
```
> npm run prettier:run -- --write

node:internal/modules/cjs/loader:959
  throw err;
  ^

Error: Cannot find module 'semver'
Require stack:
- /usr/share/npm/lib/utils/unsupported.js
- /usr/share/npm/bin/npm-cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/usr/share/npm/lib/utils/unsupported.js:2:14)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/share/npm/lib/utils/unsupported.js',
    '/usr/share/npm/bin/npm-cli.js'
  ]
}
```

**Comment by DamnClin** [comment]
Did you do an `npm i` before that ?

**Comment by biergit** [comment]
Yes. I'll try reinstalling NPM and see what happens.

**Comment by biergit** [comment]
I used the NPM installed by the frontend plugin to run `npm run prettier:format` but it failed with 
```
 > npm run prettier:run -- --write       
/usr/bin/env: ‘bash\r’: No such file or directory
```
Running `> npm run prettier:run -- --write` worked though...
Using WSL on Windows.

**Comment by pascalgrimaud** [comment]
Interesting @biergit as the hook comes from the husky.sh files, which depends on husky dependencies

**Comment by pascalgrimaud** [comment]
Just did a quick tested on my current project, and it can't work without this dialect.
So as I said, 2 possible solutions:
- keep like current today
- remove from the classic postgresql module and add this dialect in a dedicated module, as it's important to have

**Comment by biergit** [comment]
> Just did a quick tested on my current project, and it can't work without this dialect. So as I said, 2 possible solutions:
> 
>     * keep like current today
> 
>     * remove from the classic postgresql module and add this dialect in a dedicated module, as it's important to have

Could you share a minimal working example?

**Comment by biergit** [comment]
> Interesting @biergit as the hook comes from the husky.sh files, which depends on husky dependencies

For me the precommit hook with IntelliJ and code on WSL Ubuntu fails with:
> /usr/bin/env: ‘bash\r’: No such file or directory
husky - pre-commit hook exited with code 127 (error)

**Comment by DamnClin** [comment]
cc @Gnuk I know you digged a lot into the npm execution (and the maven frontend plugin). Any idea for @biergit problem?

**Comment by pascalgrimaud** [comment]
@biergit :

Here a sample project: https://github.com/pascalgrimaud/jhlite-sample-psql-lob
Here the commit, when adding a new entity: https://github.com/pascalgrimaud/jhlite-sample-psql-lob/commit/234c173c5237180aaa9ac8709e982075a2dc184f

You can try to remove the following config and you'll see it won't work:

```
spring.jpa.database-platform=com.mycompany.myapp.technical.infrastructure.secondary.postgresql.FixedPostgreSQL10Dialect
```

Here the stacktrace:

```
2022-10-29 10:48:41.425  WARN 27203 --- [           main] o.h.engine.jdbc.spi.SqlExceptionHelper   : SQL Error: 0, SQLState: 42804
2022-10-29 10:48:41.426 ERROR 27203 --- [           main] o.h.engine.jdbc.spi.SqlExceptionHelper   : ERROR: column "file" is of type bytea but expression is of type bigint
  Indice : You will need to rewrite or cast the expression.
  Position : 45

org.springframework.dao.InvalidDataAccessResourceUsageException: could not execute statement; SQL [n/a]; nested exception is org.hibernate.exception.SQLGrammarException: could not execute statement

	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:259)
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:233)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.translateExceptionIfPossible(AbstractEntityManagerFactoryBean.java:551)
	at org.springframework.dao.support.ChainedPersistenceExceptionTranslator.translateExceptionIfPossible(ChainedPersistenceExceptionTranslator.java:61)
	at org.springframework.dao.support.DataAccessUtils.translateIfNecessary(DataAccessUtils.java:242)
	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:152)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:174)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:215)
	at jdk.proxy2/jdk.proxy2.$Proxy182.findAll(Unknown Source)
	at com.mycompany.myapp.dummy.infrastructure.secondary.JpaBurgerRepositoryIT.shouldFindAll(JpaBurgerRepositoryIT.java:30)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: org.hibernate.exception.SQLGrammarException: could not execute statement
	at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:103)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:37)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:200)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:46)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3375)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3908)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:107)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:604)
	at org.hibernate.engine.spi.ActionQueue.lambda$executeActions$1(ActionQueue.java:478)
	at java.base/java.util.LinkedHashMap.forEach(LinkedHashMap.java:721)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:475)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:344)
	at org.hibernate.event.internal.DefaultAutoFlushEventListener.onAutoFlush(DefaultAutoFlushEventListener.java:57)
	at org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:107)
	at org.hibernate.internal.SessionImpl.autoFlushIfRequired(SessionImpl.java:1372)
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1452)
	at org.hibernate.query.internal.AbstractProducedQuery.doList(AbstractProducedQuery.java:1649)
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1617)
	at org.hibernate.query.Query.getResultList(Query.java:165)
	at org.hibernate.query.criteria.internal.compile.CriteriaQueryTypeQueryAdapter.getResultList(CriteriaQueryTypeQueryAdapter.java:76)
	at org.springframework.data.jpa.repository.support.SimpleJpaRepository.findAll(SimpleJpaRepository.java:427)
	at org.springframework.data.jpa.repository.support.SimpleJpaRepository.findAll(SimpleJpaRepository.java:88)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.data.repository.core.support.RepositoryMethodInvoker$RepositoryFragmentMethodInvoker.lambda$new$0(RepositoryMethodInvoker.java:289)
	at org.springframework.data.repository.core.support.RepositoryMethodInvoker.doInvoke(RepositoryMethodInvoker.java:137)
	at org.springframework.data.repository.core.support.RepositoryMethodInvoker.invoke(RepositoryMethodInvoker.java:121)
	at org.springframework.data.repository.core.support.RepositoryComposition$RepositoryFragments.invoke(RepositoryComposition.java:530)
	at org.springframework.data.repository.core.support.RepositoryComposition.invoke(RepositoryComposition.java:286)
	at org.springframework.data.repository.core.support.RepositoryFactorySupport$ImplementationMethodExecutionInterceptor.invoke(RepositoryFactorySupport.java:640)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.doInvoke(QueryExecutorMethodInterceptor.java:164)
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.invoke(QueryExecutorMethodInterceptor.java:139)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.data.projection.DefaultMethodInvokingMethodInterceptor.invoke(DefaultMethodInvokingMethodInterceptor.java:81)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:123)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:388)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:137)
	... 77 more
Caused by: org.postgresql.util.PSQLException: ERROR: column "file" is of type bytea but expression is of type bigint
  Indice : You will need to rewrite or cast the expression.
  Position : 45
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:355)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:167)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:135)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
```

**Comment by biergit** [comment]
@pascalgrimaud It should work when you remove the `@Lob` annotation from the `byte[]` attribute.

**Comment by pascalgrimaud** [comment]
Because my applications were a mixt of classic generator-jhipster apps and jhlite apps, and we was still using `@Lob`.
That's why in generator-jhipster, there is still this Dialect

And for this reason, I suggested to create a dedicated module for keeping this behavior

It could be done in this PR or in a new one, but it's important to keep this class somewhere IMO :-D

**Comment by biergit** [comment]
Could you explain how backwards compatibility is a concern in this project? I thought the usecase was to generate a scaffold and go - no updates. 
Also if backwards compatibility is a concern then the Spring Boot 3 update should break a lot of things anyway.

**Comment by DamnClin** [comment]
> Could you explain how backwards compatibility is a concern in this project?

For JHLite (let's say internal / motor) we need release - 2 because of `custom-jhlite` allowing users to to deploy there own JHLite instance with custom modules. So we are doing -> deprecate -> delete after 2 releases (2 because of 1 version shift while using the JHLite dependency).

For the generated app this is a different story. Generator JHipster still have way more users so we are trying to support the feature from this generator since it's important for users. You can totally split one feature from generator-jhipster in multiple modules here but we are trying to support "infrastructure" stuff from generator. Supporting JDL is a totally different story :D (see https://github.com/jhipster/jhipster-lite/discussions/2754)

**Comment by Gnuk** [comment]
> cc @Gnuk I know you digged a lot into the npm execution (and the maven frontend plugin). Any idea for @biergit problem?

Maybe it's because Windows line break is `\r\n` so when it's launched in WSL the program want to launch `bash\r` instead of `bash`.

The idea I have on this may be the `core.autocrlf` that can be put to `input` instead of `true` in order to get `\n` from the repo. The other problem with this option is that the `\n` from the repo will be received on the next file update or on a `clone`.

**Comment by pascalgrimaud** [comment]
> Could you explain how backwards compatibility is a concern in this project? I thought the usecase was to generate a scaffold and go - no updates.
Also if backwards compatibility is a concern then the Spring Boot 3 update should break a lot of things anyway.

No, don't worry about backwards compatiblity, it's just a feedback from my use :-D
Are you ok with my suggestions ?
- remove from the classic postgresql module (your current PR)
- add this dialect in a dedicated module (a new one, I can take care of it, probably next week)

**Comment by biergit** [comment]
Mhh, I don't fully understand what you mean by
> [...] so we are trying to support the feature from this generator
Could you give an example?
Also you have quite a large API surface here. Even doing something like [this](https://github.com/jhipster/jhipster-lite/pull/4166#issuecomment-1295777669) is a breaking change.

**Comment by DamnClin** [comment]
It is a best effort :D, if something breaking happens it's OK (for now)

**Comment by biergit** [comment]
> > Could you explain how backwards compatibility is a concern in this project? I thought the usecase was to generate a scaffold and go - no updates.
> > Also if backwards compatibility is a concern then the Spring Boot 3 update should break a lot of things anyway.
> 
> No, don't worry about backwards compatiblity, it's just a feedback from my use :-D Are you ok with my suggestions ?
> 
>     * remove from the classic postgresql module (your current PR)
> 
>     * add this dialect in a dedicated module (a new one, I can take care of it, probably next week)

Sure if you think that anyone really cares about that old dialect. To me it just looks like dead code.

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-import tech.jhipster.lite.TestFileUtils;
-import tech.jhipster.lite.UnitTest;
-import tech.jhipster.lite.module.domain.JHipsterModule;
-import tech.jhipster.lite.module.domain.JHipsterModulesFixture;
-import tech.jhipster.lite.module.domain.docker.DockerImageVersion;
-import tech.jhipster.lite.module.domain.docker.DockerImages;
-import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+import org.junit.jupiter.api.*;
```
Comment (pascalgrimaud) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java:6]: Can you fix these import plz?

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**